docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -274,6 +293,24 @@ To add new subscription rules and edit existing ones, complete the following ste
 
 If you plan to use the API to add products to the subscription sales channel, see [product channel assignments](/api-docs/multi-storefront/api-guide#products) for more information.
 
+## Deploying with Vercel
+
+To deploy with Vercel, click on the **Deploy** button in the [README.md](https://github.com/bigcommerce/subscription-foundation/blob/main/README.md) file, login to GitHub if you have not done so, and complete the following steps:
```

Comment:
Replace the first sentence with a brief intro: "The BigCommerce Subscription Foundation framework enables you to deploy your application with Vercel directly from the GitHub repo. 

To deploy your app with Vercel:

1. Log into GitHub, and then navigate to the [BigCommerce Subscription Foundation Readme]((https://github.com/bigcommerce/subscription-foundation/blob/main/README.md) file.
2. Scroll down the page to the **Deploy with Vercel** section and click **Deploy**.
3. In the **Get started** section, select a Git provider, and then authorize Vercel to connect to your provider.
4. In the **Create Git Repository** section, select a Git scope, enter a name for the repository, and then click **Create**.
5. (Optional) If you are deploying to Vercel from a GitHub organization, you need to create a team by following the steps in the **Create a team** section.
6. In the **Configure Project** section, in the **Database_URL** field enter the connection string for the Supabase database you created in the previous step. See..... 
7. Enter '1234' for the remaining environment variables, and then click **Deploy**. The deployment process will take a few minutes.
8. After you have successfully deployed Vercel, it is recommended that you update your environment variables. For more information, see....

Reply:
@stanzikratelbc Working on getting a screenshot for item #5